### PR TITLE
Merge answers detail view

### PR DESCRIPTION
**Pull request by mehalshah**
Multiple changes to the detail view

- [ ] Add support for showing multiple answers in one row in the detail view
- [ ] Gender Identity and Race should show up in the first table, but should still be asked in the 5th part of the teacher application. Remove the demographic section from the detail view
- [ ] Miscellaneous copy changes
- [ ] Move principal approval state strings into partial consts
- [ ] Change autoscoring consts
- [ ] Handle not having school stats data in application serializer
- [ ] Change the serializer to have flatter racial data to make it easier for the serializer to access

### CODE REVIEW

File: dashboard/app/models/pd/application/teacher1920_application.rb [hunk 1]
```diff
@@ -501,19 +521,28 @@ def meets_criteria
...
-      scores = (response_scores_hash[:meets_scholarship_criteria_scores] || {}).values
-
-      if scores.uniq == [YES]
-        YES
-      elsif NO.in? scores
-        NO
+      if principal_approval_state == NOT_REQUIRED
+        # If there is no needed principal approval, then criteria is just whether
+        # the one scholarship question is yes
+        response_scores_hash[:meets_scholarship_criteria_scores][:previous_yearlong_cdo_pd] || 'Reviewing Incomplete'
```
Comment (clareconstantine) [inline dashboard/app/models/pd/application/teacher1920_application.rb]: use the `REVIEWING_INCOMPLETE` constant here

File: dashboard/test/models/pd/application/teacher1920_application_test.rb [hunk 2]
```diff
@@ -75,7 +75,7 @@ class Teacher1920ApplicationTest < ActiveSupport::TestCase
           committed: 'Yes'
         }
       }.to_json
-      assert_equal 'Reviewing incomplete', teacher_application.meets_criteria
+      assert_equal 'Reviewing Incomplete', teacher_application.meets_criteria
```
Comment (clareconstantine) [inline dashboard/test/models/pd/application/teacher1920_application_test.rb]: Use `REVIEWING_INCOMPLETE` constant

File: lib/cdo/shared_constants/pd/teacher1920_application_constants.rb [hunk 3]
```diff
@@ -185,20 +225,18 @@ def self.clean_multiline(string)
         :csd_which_grades,
         :cs_total_course_hours,
         :plan_to_teach,
-        :have_cs_license,
         :committed,
         :willing_to_travel,
-        :principal_approval
+        :principal_implementation
```
Comment (clareconstantine) [inline lib/cdo/shared_constants/pd/teacher1920_application_constants.rb:312]: just making sure I understand - including `:principal_implementation` in this list won't affect the score if principal approval is not required and the question isn't answered?

Comment (mehalshah) [reply lib/cdo/shared_constants/pd/teacher1920_application_constants.rb:312]: Correct. Principal implementation isn't used for judging criteria if the approval is unrequired

**Pull request merged**